Sysdig - update team event settings (#785)
BCDevOps/platform-services: monitoring/sysdig/docs/sysdigteam_user_guide.md

  oc project 101ed4-tools
  oc apply -f sysdigteam-sample.yml
  ```
- Validate the creation of the Sysdig Team using `oc describe sysdig-team <your_sysdig_team_cr_name>`
  ```shell
  Name:         101ed4-sysdigteam
  Namespace:    101ed4-tools
  Labels:       <none>
  API Version:  ops.gov.bc.ca/v1alpha1
  Kind:         SysdigTeam
  Metadata:
    Creation Timestamp:  2021-04-15T22:42:20Z
    ...
  Spec:
    Team:
      Description:  The Sysdig Team for the Platform Services Documize
      Users:
        Name:  shelly.han@gov.bc.ca
        Role:  ROLE_TEAM_MANAGER
        Name:  patricksimonian@gmail.com
        Role:  ROLE_TEAM_EDIT
        ...
  Status:
    Conditions:
      Ansible Result:
        Changed:             0
        Completion:          2021-08-18T20:10:43.665524
        Failures:            0
        Ok:                  30
        Skipped:             13
      Last Transition Time:  2021-08-05T18:54:24Z
      Message:               Awaiting next reconciliation
      Reason:                Successful
      Status:                True
      Type:                  Running
  Events:                    <none>

  ```

> > when you see `Awaiting next reconciliation` and Successful status, that means the sysdig team has been created for you all good. But if you see other status message, please contact us on RocketChat channel https://chat.developer.gov.bc.ca/channel/devops-sysdig!

Reviewer: ```suggestion
> when you see `Awaiting next reconciliation` and Successful status, that means the sysdig team has been created successfully. If you do not see `Awaiting next reconciliation`, please contact us on RocketChat channel https://chat.developer.gov.bc.ca/channel/devops-sysdig!
```
